package/MDAnalysis/topology/PDBParser.py

```diff
@@ -344,10 +336,16 @@ def _parseatoms(self):
                     wmsg = (f"Unknown element {elem} found for some atoms. "
                             f"These have been given an empty element record. "
                             f"If needed they can be guessed using "
-                            f"MDAnalysis.topology.guessers.")
+                            f"universe.guess_TopologyAttributes(context='PDB',"
```

Comment:
The "PDB" context does not currently exist yet, right? If the default context gives the right behaviour, perhaps we should recommend that instead here?